chore: Replace jQuery UI datepicker with flatpickr

labmecanicatec opened the pull request:
Switched the datepicker implementation from jQuery UI to flatpickr for improved UI and maintainability. Added flatpickr assets, updated styles in librebooking.css, removed jQuery UI datepicker-specific styles from schedule.css, and updated relevant templates and JavaScript to use flatpickr.

JohnVillalovos commented on Web/scripts/schedule.js:
```diff
@@ -1198,27 +1201,44 @@ function ChangeGroup(node) {
+
     if ($('#multidateselect').is(':checked')) {
-        AddSpecificDate(dateText, inst);
+        AddSpecificDate(date, inst);
```
@labmecanicatec Can this be fixed? Besides that I don't have any objections to this. I tested it locally and looks good. Thanks.

<img width="1920" height="1394" alt="image" src="https://github.com/user-attachments/assets/b84e3724-0bd7-4486-870b-00fdef4a9987" />

labmecanicatec committed:
chore: Replace jQuery UI datepicker with flatpickr

Switched the datepicker implementation from jQuery UI to flatpickr for
improved UI and maintainability. Added flatpickr assets, updated styles
in librebooking.css, removed jQuery UI datepicker-specific styles from
schedule.css, and updated relevant templates and JavaScript to use
flatpickr.
